sycl/doc/extensions/supported/sycl_ext_oneapi_backend_level_zero.md

```diff
@@ -389,6 +412,67 @@ Construct a SYCL buffer instance from a pointer to a Level Zero memory allocatio
 description above for semantics and restrictions.
 The additional <code>AvailableEvent</code> argument must be a valid SYCL event. The instance of the SYCL buffer class template being constructed must wait for the SYCL event parameter to signal that the memory native handle is ready to be used.
 </tr>
+
+<tr>
+<td>
+
+``` C++
+make_image<backend::ext_oneapi_level_zero, Dims>(

```

Comment:
This does not address my previous comment about showing the template parameters.  Because `make_image` is not in the SYCL 2020 specification, there is no specification of the template parameters or their default values.  Even the return type of this function is not specified.

An additional problem is that all of the entries in this table are a sort of pseudo code.  They aren't template definitions because they don't show the `tempalate` keyword and template arguments.  They are sort of a pseudo-code version of a template specialization, but the syntax isn't correct.

I think the best solution is to change all the entries in the first column to be real template definitions.  Your entries for `make_image` would look like:

```
template<backend Backend, int Dimensions = 1, typename AllocatorT = sycl::image_allocator>
image<Dimensions, AllocatorT> make_image(
    const backend_input_t<Backend, image<Dimensions, AllocatorT>> &backendObject,
    const context &targetContext);

template<backend Backend, int Dimensions = 1, typename AllocatorT = sycl::image_allocator>
image<Dimensions, AllocatorT> make_image(
    const backend_input_t<Backend, image<Dimensions, AllocatorT>> &backendObject,
    const context &targetContext, event availableEvent);
```

(I'm guessing at the default template parameters here, so correct them if necessary.)

To make the entries in the table consistent, you should also change the other entries to be real template definitions.  The synopses can just be copied from the SYCL 2020 specification.

Reply:
done